.github/workflows/checks.yaml

```diff
@@ -0,0 +1,32 @@
+name: Checks
+
+on:
+  push:
+    branches: [main]
```

Comment:
If we're merging this to the 6.x branch, we should either change this to include both main and 6.x or just 6.x and we can change this field in main

Reply:
Ah shoot, thank you! I changed it so it included both main and 6.x but I'm open to having just 6.x if that's preferred. I used the syntax used [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-filters) but I wasn't 100% sure so I used act to test it again and the job succeeded